media-sound/supercollider/supercollider-3.8.0.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+# are all of theses needed ? 
+inherit cmake-utils eutils multilib versionator
```

Comment:
I don't see where you need `eutils`, `multilib` or `versionator`. You will probably need `readme.gentoo-r1` though